sklearn/utils/estimator_checks.py

```diff
@@ -688,7 +688,9 @@ def check_transformers_unfitted(name, transformer):
     X, y = _boston_subset()
 
     transformer = clone(transformer)
-    assert_raises((AttributeError, ValueError), transformer.transform, X)
+    with assert_raises((AttributeError, ValueError),
+                       msg="Transformers unfitted"):
```

Comment:
Message should be more like: "The unfitted transformer {name} does not raise an error when `transform` is called. Perhaps use check_is_fitted"